src/Workspaces/Core/Portable/Workspace/Solution/DocumentState.cs

```diff
@@ -559,6 +559,15 @@ public bool TryGetSyntaxTree(out SyntaxTree syntaxTree)
             return treeAndVersion.Tree;
         }
 
+        internal SyntaxTree GetSyntaxTree(CancellationToken cancellationToken)
```

Comment:
Not to step into the "what should the visibility of members in a non-public type" debate, it is odd this one is internal and the other is public.